client/html/src/Client/Html/Account/Profile/Account/Standard.php

```diff
@@ -0,0 +1,264 @@
+    {
+        $view = $this->getView();
+
+        $html = '';
+        foreach( $this->getSubClients() as $subclient ) {
+            $html .= $subclient->setView( $view )->getBody( $uid );
+        }
+        $view->accountBody = $html;
+
+        /** client/html/account/profile/account/template-body
+         * Relative path to the HTML body template of the account profile account client.
+         *
+         * The template file contains the HTML code and processing instructions
+         * to generate the result shown in the body of the frontend. The
+         * configuration string is the path to the template file relative
+         * to the templates directory (usually in client/html/templates).
+         *
+         * You can overwrite the template file configuration in extensions and
+         * provide alternative templates. These alternative templates should be
+         * named like the default one but with the string "standard" replaced by
+         * an unique name. You may use the name of your project for this. If
+         * you've implemented an alternative client class as well, "standard"
+         * should be replaced by the name of the new class.
+         *
+         * @param string Relative path to the template creating code for the HTML page body
+         * @since 2019.07
+         * @category Developer
+         * @see client/html/account/profile/account/template-header
+         */
+        $tplconf = 'client/html/account/profile/account/template-body';
+        $default = 'account/profile/account-body-standard';
+
+        return $view->render( $view->config( $tplconf, $default ) );
+    }
+
+    /**
+     * Processes the input, e.g. store given values.
+     *
+     * A view must be available and this method doesn't generate any output
+     * besides setting view variables if necessary.
+     * @throws \Aimeos\Client\Html\Exception|\Aimeos\Controller\Frontend\Exception
+     */
+    public function process()
+    {
+        $view = $this->getView();
+
+        if( !$view->param( 'account/save' , false) ) {
+            parent::process();
+            return;
+        }
+
+        /** @var \Aimeos\Controller\Frontend\Customer\Standard $cntl */
+        $cntl = \Aimeos\Controller\Frontend::create( $this->getContext(), 'customer' );
+        $oldPassword = $cntl->get()->getPassword();
+        $values = $view->param('account', []);
+
+        $isNew = $values['customer.newpassword'] !== $values['customer.oldpassword'];
+        $confirmed = $values['customer.newpassword'] === $values['customer.confirmnewpassword'];
+
+        $errors = [];
+
+        $isValid = true;
+        try {
+            Validator::make([
+                'password' => $values['customer.newpassword'],
+                'password_confirmation' => $values['customer.confirmnewpassword']
+            ], [
+                'password' => ['required', 'string', new Password, 'confirmed']
+            ])->validate();
```

Comment:
You can't use Laravel specific code in HTML clients. This won't work in other integrations.